gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -264,7 +264,13 @@ nnsconf_loadconf (gboolean force_reload)
   }
 
   /* Read from the conf file first */
-  conf.conffile = g_strdup (NNSTREAMER_CONF_FILE);
+  if (NNSTREAMER_CONF_FILE[0] != '/') {
+    /** default value of 'sysconfdir' in meson is 'etc' */
+    conf.conffile = g_strdup_printf("/%s", NNSTREAMER_CONF_FILE);
```

Comment:
please consider windows

Reply:
Can we check and change this path in meson script?

Reply:
@jaeyun-jung https://developer.gnome.org/glib/stable/glib-Miscellaneous-Utility-Functions.html#g-path-is-absolute

Reply:
@jaeyun-jung IMO, g_path_is_absolute can cover both windows and linux environments.

Reply:
But, ```"/%s"``` can't.. can it?

Reply:
And.. how does meson's sysconfdir affect this?

Reply:
meson default ```sysconfdir``` value now changed, from ```/etc``` to ```etc```.

Reply:
> But, `"/%s"` can't.. can it?
@myungjoo No, as you said, it can't.

> And.. how does meson's sysconfdir affect this? Can you supply an input that creates the issue you mentioned?
@myungjoo 
```bash
$ meson build
The Meson build system
Version: 0.50.999
Source dir: /home/wook/Work/prj_nnstreamer/nnstreamer
Build dir: /home/wook/Work/prj_nnstreamer/nnstreamer/build
Build type: native build
Project name: nnstreamer
Project version: 0.1.3
Native C compiler: ccache cc (gcc 5.4.0 "cc (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609")
Native C++ compiler: ccache c++ (gcc 5.4.0 "c++ (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609")
Build machine cpu family: x86_64
Build machine cpu: x86_64
...
Dependency libpng found: YES 1.2.54
Program cp found: YES (/bin/cp)
Dependency GTest found: YES 1.9.0
Configuring nnstreamer-test.ini using configuration
Configuring nnstreamer.ini using configuration
Configuring nnstreamer.pc using configuration
Build targets in project: 32
Found ninja-1.8.2 at /usr/bin/ninja

$ cd build
$ ninja
...
[3/104] ccache cc -Igst/nnstreamer/73d9e45@@nnstreamer@sha -Igst/nnstreamer -I../gst/nnstreamer ... 
'-DNNSTREAMER_CONF_FILE="etc/nnstreamer.ini"' 
...
...
```

Reply:
> meson default `sysconfdir` value changed, from `/etc` to `etc`.
> 
> now nnstreamer sets ini install dir from this.
> nnstreamer_inidir = get_option('sysconfdir')
> 
> how about changing this? (like other options)
> nnstreamer_libdir = join_paths(nnstreamer_prefix, get_option('libdir'))
> nnstreamer_inidir = join_paths(nnstreamer_prefix, get_option('sysconfdir'))
> 
> if user sets --prefix=/usr --sysconfdir=/etc, nnstreamer_inidir=/etc
> if user sets --prefix=/usr --sysconfdir=etc (or use default), nnstreamer_inidir=/usr/etc

@jaeyun-jung 
IMO, if user sets --prefix=/usr --sysconfdir=/etc, nnstreamer_inidir would be /usr//etc rather than /etc.

Reply:
@wooksong It is /etc. join_paths() ignores old params if having absolute path.
(https://mesonbuild.com/Reference-manual.html#join_paths)

Reply:
@jaeyun-jung Oh, I see. I have submitted another version of this PR. How about the way I used in another PR?